src/components/BlockImage.js

```diff
@@ -26,10 +27,19 @@ const BlockImage = props => {
         marginBottom: `${marginBottom}rem`,
         marginTop: `${marginTop}rem`
       }}
-      className="BlockImage case-study-block-container"
+      className={cx("BlockImage p1 flex", {
+        'justify-center': imageAlign === 'center',
+        'justify-start': imageAlign === 'left',
+        'justify-end': imageAlign === 'right',
+      })}
     >
       <div
-        className={`BlockImage__image-container BlockImage__image-container--${imageSize}-${imageAlign} overflow-hidden flex items-center justify-center`}
+        className={cx(`BlockImage__image-container overflow-hidden`, {
+          'md:col-12': imageSize === 'full',
+          'md:col-6': imageSize === 'large',
+          'md:col-5': imageSize === 'medium',
+          'md:col-4': imageSize === 'small'
+        })}
```

Comment:
I see you're using backticks, single quotes, and double quotes all the same way in a matter of 7 lines lol. I recommend sticking with single quotes or installing prettier to do that for ya :P

Reply:
oh, I thought there is prettier already so I didn't pay attention too much! I will install it!